Prac_06/guitar.py

```diff
@@ -0,0 +1,16 @@
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+        self.age = 0
+
+    def __str__(self):
+        return "{} ({}) : ${}".format(self.name, self.year, self.cost)
+
+    def get_age(self):
+        self.age = 2020 - self.year
```

Comment:
Perhaps change '2020' to a constant at the top of the file, such as:
`PRESENT_YEAR = 2020`